backend/app/engine/projects/rewards/__init__.py

```diff
@@ -26,21 +27,24 @@ def __iter__(self):
 @dataclass
 class ProjectRewardsPayload:
     matched_rewards: int = None
-    allocations: List[AllocationItem] = None
+    before_allocations: List[AllocationItem] = None
     projects: List[str] = None
+    user_new_allocations: List[AllocationItem] = field(default_factory=list)
 
 
 @dataclass
 class ProjectRewardsResult:
     rewards: List[ProjectRewardDTO]
     rewards_sum: int
     total_allocated: int
+    leverage: float
```

Comment:
what does leverage mean for a project?

Reply:
ProjectRewardsResult is a payload returned for the global computations for all projects, that's why leverage is put here next to total_allocated, rewards_sum etc.

Reply:
ok cool